src/js/components/Grommet/Grommet.js

```diff
@@ -1,14 +1,13 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-import deepAssign from 'deep-assign';
-import cloneDeep from 'clone-deep';
 
 import StyledGrommet from './StyledGrommet';
 
-import baseTheme from '../../themes/vanilla';
-
 import doc from './doc';
 
+import baseTheme from '../../themes/vanilla';
```

Comment:
I was trying to follow the pattern of putting imports that are further away earlier than imports that are closer.